README.md

```
@@ -1,177 +1,283 @@
+
+> "What's the current tuning system? Is Ableton Link enabled?"
+
+> "Control the Looper — start recording, then overdub"
+
+> "Show me the warp markers on this audio clip and adjust the timing"
+
+---
+
+## Architecture
+
+```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)